pkg/service/backup/worker_clustercheck.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2022 ScyllaDB
+
+package backup
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/scylladb/scylla-manager/v3/pkg/util/retry"
+)
+
+// CheckCluster checks the current load on every element of hosts and returns the first one exceeding the configured
```

Comment:
> every element of hosts and returns the first one exceeding the configured maximum threshold

This comment sounds to me as if this function was supposed to return an actual node / load, not just validation error.
Maybe change the comment a little bit so that it indicates validation in a more direct way.

Reply:
Agree, it's on me actually, I suggested that comment 😅